safe-global/safe-wallet-web
# feat: Change threshold flow

## What it solves

Part of #2067 

## How this PR fixes it

- Moves the Change Threshold to the new tx flow
- Fixes a bug regarding gasLimit estimation in the SignOrExecuteForm

## How to test it

1. Open the Settings page on a 2/n safe
2. Change the threshold
3. Observe the new tx flow

## Screenshots
<img width="1191" alt="Screenshot 2023-06-13 at 11 21 17" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/b94b4095-9316-4cdf-b3d6-3294530fdf11">

## Checklist
